src/mumble/AudioOutput.cpp

```diff
@@ -128,8 +132,8 @@ float AudioOutput::calcGain(float dotproduct, float distance) {
 }
 
 void AudioOutput::wipe() {
-	foreach (AudioOutputUser *aop, qmOutputs)
-		removeBuffer(aop);
+	foreach (AudioOutputBuffer *buffer, qmOutputs)
+		removeBuffer(buffer);
```

Comment:
```suggestion
	for (AudioOutputBuffer *buffer : qmOutputs) {
		removeBuffer(buffer);
	}
```
While changing these lines anyway, we might as well get rid of Qt-specific `foreach`